.gitignore

```diff
@@ -59,5 +59,7 @@ build-aux/compile
 build-aux/test-driver
 libsecp256k1.pc
 
+### CMake
+CMakeUserPresets.json
```

Comment:
> > Maybe add `/CMakeUserPresets.json` to the `.gitignore`?
> 
> Done. I took the version without the leading slash because this file could in principle exist in `src/` or `examples/` too.

https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html:
> `CMakePresets.json` and `CMakeUserPresets.json` live in the project's root directory.

Reply:
Ok, you're right, fixed.